static/js/components/tiles/answer_table_tile.tsx

```diff
@@ -149,14 +149,15 @@ const fetchData = async (
     propResp.forEach((resp, i) => {
       Object.keys(resp.data).forEach((entity) => {
         const val = resp.data[entity];
-        const entityResults = [];
+        const entityResults = new Set();
         val.forEach((singleVal) => {
-          entityResults.push(
+          entityResults.add(
             singleVal.name || singleVal.value || singleVal.dcid
           );
           provIds.add(singleVal.provenanceId);
         });
-        values[entity][props.columns[i].propertyExpr] = entityResults;
+        values[entity][props.columns[i].propertyExpr] =
+          Array.from(entityResults);
```

Comment:
Does order matter?

Reply:
no, I don't think the order matters when displaying the values